src/utils/callCRMAPI.js

```diff
@@ -2,38 +2,48 @@ export async function loadRoundRobinSettings() {
   await window.ZOHO.embeddedApp.init();
 
   const roundRobinSettingResponse = await window.ZOHO.CRM.API.getAllRecords({
-    Entity: "advancedroundrobin__Round_Robin_Settings"
+    Entity: "advancedroundrobin__Round_Robin_Settings",
   });
 
   if (!roundRobinSettingResponse.data) {
     return [];
   }
 
-  const {
-    data: roundRobinSettings
-  } = roundRobinSettingResponse;
+  const { data: roundRobinSettings } = roundRobinSettingResponse;
 
   return roundRobinSettings.map((roundRobinSetting) => {
-    const ownerName = roundRobinSetting['Owner']?.name ?? roundRobinSetting['advancedroundrobin.Owner']?.name ?? roundRobinSetting['advancedroundrobin.advancedroundrobin.Owner']?.name;
+    const ownerName =
+      roundRobinSetting["Owner"]?.name ??
+      roundRobinSetting["advancedroundrobin.Owner"]?.name ??
+      roundRobinSetting["advancedroundrobin.advancedroundrobin.Owner"]?.name;
     return {
       ...roundRobinSetting,
       ownerName,
-      key: roundRobinSetting.id
+      key: roundRobinSetting.id,
     };
   });
 }
 
 export async function loadRoundRobinSetting(recordID) {
   await window.ZOHO.embeddedApp.init();
 
-  const {
-    data: roundRobinSetting
-  } = await window.ZOHO.CRM.API.getRecord({
+  const { data: roundRobinSetting } = await window.ZOHO.CRM.API.getRecord({
     Entity: "advancedroundrobin__Round_Robin_Settings",
-    RecordID: recordID
+    RecordID: recordID,
   });
 
-  return roundRobinSetting[0];
+  const roundRobinAvailability = await loadRoundRobinAvailability();
+
+  const isLookup = roundRobinAvailability.filter(
```

Comment:
this doesn't seem like a good variable name. The is prefix implies it is a boolean. It's not. perhaps ```availabilityRecordsForThisSetting```